rootston/seat.c

```diff
@@ -717,6 +719,11 @@ struct roots_seat_view *roots_seat_view_from_view(
 }
 
 void roots_seat_set_focus(struct roots_seat *seat, struct roots_view *view) {
+	if (view && seat->exclusive_client && wl_resource_get_client(
+				view->wlr_surface->resource) != seat->exclusive_client) {
```

Comment:
Maybe adding a `roots_seat_accept_input(struct wl_resource *resource)` could be better